### ISSUE TITLE
older rom is favored

### ISSUE THREAD

**Issue by AlvaroBurnett** [andrebrait/1g1r-romset-generator#30]
When processing Genesis roms, the generator thinks that:

Lost Vikings, The (USA) (November, 1993)

is better than:

Lost Vikings, The (USA) (October, 1995)

**Comment by andrebrait** [comment]
It currently doesn't have a distinction for dates. I can add it, though, as long as there's a formalized format for it.

**Comment by unexpectedpanda** [comment]
No-Intro is a bit inconsistent on dates I'm afraid. So far I've had to support:

* mm-dd-yyyy
* mm-dd-yy
* month  name, yyyy

**Comment by andrebrait** [comment]
@unexpectedpanda username checks out :D

Yeah, I can introduce support for that. It's not that hard. Just a few more regexes and one term in the comparation tuple.

However, I think it should come after any sort of versioning, if one is present, so v2 is preferred over v1 February, which is preferred over v1 January, for example.

Does that make sense?

**Comment by unexpectedpanda** [comment]
Out of nowhere :P

Yep. Versions/revisions are much higher in my parent selection process than date, so makes sense to me :)

**Comment by andrebrait** [comment]
Ok, it's been a while but I'm finally gonna tackle this one this week
